test/WebJobs.Script.Tests/BlobLeaseManagerTests.cs

```diff
@@ -311,7 +278,7 @@ public void AcquiringLease_WithServerError_LogsAndRetries()
             Assert.Equal(TraceLevel.Info, acquisitionEvent.Level);
 
             TraceEvent renewalEvent = traceWriter.Traces.Skip(1).First();
-            Assert.Contains("Failed to renew host lock lease", renewalEvent.Message);
+            Assert.Contains("Failed to renew host lock lease: Another host has acquired the lease. The last successful renewal completed at ", renewalEvent.Message);
```

Comment:
For formatted messages like this, you might consider using regex for the validation, like I've done in other tests (e.g. [here](https://github.com/Azure/azure-webjobs-sdk-script/blob/dev/test/WebJobs.Script.Tests/WebScriptHostManagerTests.cs#L346)).